frontend/components/dashboard/table/SummaryDetails.vue

```diff
@@ -1,60 +1,77 @@
 <script setup lang="ts">
 import type { VDBSummaryTableRow } from '~/types/api/validator_dashboard'
-import { type SummaryDetailsEfficiencyCombinedProp, type SummaryRow } from '~/types/dashboard/summary'
```

Comment:
A small detail on mobile:
![image](https://github.com/gobitfly/beaconchain/assets/150015231/20e7afe7-e2e7-44aa-8dcd-1e91d97cb021)
the tip of the bubble points to the void. Is there a way to move it above the `i`?

Reply:
Yes this is a minor Bug in the Tooltip component, as the tooltip tries to center itself over the parent but ensures it stays visible. The small triangle is just centered relative to the tooltip and does not consider when the tooltip has to move. 

As this is a general Bug and not related to this PR, I would prefer to fix that separately. Would be cool if you could create a Bug for it so it does not get forgotten.